src/iidm/ExtensionProviders.cpp

```diff
@@ -18,10 +18,17 @@
 #include <powsybl/logging/LoggerFactory.hpp>
 #include <powsybl/stdcxx/format.hpp>
 
+#include "extensions/LoadDetailXmlSerializer.hpp"
```

Comment:
Maybe we could simply include like this
```
#include "LoadDetailXmlSerializer.hpp"
```
(to avoid the prerequisite of the include compile flag)